cardano-db-sync/src/Cardano/DbSync/Tracing/ToObjectOrphans.hs

```diff
@@ -6,12 +6,12 @@
 
 module Cardano.DbSync.Tracing.ToObjectOrphans () where
 
-import           Data.Text (Text)
+import           Data.Text (Text, pack)
```

Comment:
Please use `Text.pack` instead of the unqualified name (as per the rest of this project).